website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,154 @@
+import CreateCompoundIndex from '!!raw-loader!../../guides/create-indexes/3-create-compound-index.request.js';
+import CreatePartialIndex from '!!raw-loader!../../guides/create-indexes/4-create-partial-index.request.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Create one or more indexes on a given collection to improve query performance or enable certain query features (e.g., full-text, vector, and TTL indexes).
+
+<!-- Syntax structure of the command here if there are no variations, types, or usage examples to command. Otherwise, use syntax inside subcategories-->
+
+<!-- markdownlint-disable MD033 -->
+
+This command accepts a collection name and an array of index specification documents.
+The index specification document specifies the fields to be indexed, the index name, and other options such as uniqueness, partial filter expressions, and expiration times for TTL indexes.
+
+<!-- majorly fields such as arguments, modifers, stage, operators, parameters, options supported by the command in a table (if necessary or available). Takes in the following parameters: -->
+
+| Field                     | Type     | Description                                                                                                                                       |
+| ------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `createIndexes`           | string   | The name of the collection.                                                                                                                       |
+| `indexes`                 | array    | An array of index specification documents. Each document defines an index.                                                                        |
+| `key`                     | document | A single document that contains the field(s) to index and their sort order or type (`1`, `-1`, `"text"`, `cosmosSearch` for vector indexes, etc). |
+| `name`                    | string   | A required name for the index. Must be unique to the collection.                                                                                  |
+| `unique`                  | boolean  | Ensures all values are unique for the indexed field if set to `true`.                                                                             |
+| `partialFilterExpression` | document | Restricts index to documents matching the condition.                                                                                              |
+| `sparse`                  | boolean  | Only indexes documents that contain the indexed field if set to `true`.                                                                           |
+
+## Example usage
```

Comment:
The previous reference pages had more consistent format like https://docs.ferretdb.io/reference/operators/query/bitwise-operators/. The format for this page is quite different :thinking:

Reply:
Yea, we are trying to have a more consistent format now going forward, with changes coming to those pages as well.